packages/py/python-websocket-client/pspec_x86_64.xml

```diff
@@ -57,12 +58,12 @@
         </Files>
     </Package>
     <History>
-        <Update release="14">
-            <Date>2024-02-15</Date>
+        <Update release="15">
+            <Date>2024-06-30</Date>
             <Version>1.6.1</Version>
             <Comment>Packaging update</Comment>
-            <Name>Joey Riches</Name>
-            <Email>josephriches@gmail.com</Email>
+            <Name>Clifford Red</Name>
+            <Email>A108384@GMAIL.COM</Email>
```

Comment:
Why this is all capital letter?

Reply:
Do I actually need to change the capitalization of my email address or is this just a question?
If I need to change it, where does this come from?  When I rebuild the package it is updated automatically.

Also - can you tell me the best way to consolidate these commits?

Sorry - I am still trying to figure out git and github.   I would appreciate your patience.